src/jukebox/components/rfid/hardware/rc522_spi/requirements.txt

```diff
@@ -1,13 +1,6 @@
 # RC522 related requirements
 # You need to install these with `sudo python3 -m pip install --upgrade --force-reinstall -q -r requirements.txt`
 
-# pi-rc522 use latest version from Github
-# This is the original versions. Seems unmaintained for the moment
-# git+https://github.com/ondryaso/pi-rc522.git#egg=pi-rc522
-
-# The fork of kevinvalk has some good improvements
-# https://github.com/kevinvalk/pi-rc522
-# Get the kevinvalk fork yet again from a different fork which ensures compatibility with the Phoniebox
-git+https://github.com/ChisSoc/pi-rc522.git#egg=pi-rc522
+pi-rc522==2.3.0
```

Comment:
> # Get the kevinvalk fork yet again from a different fork which ensures compatibility with the Phoniebox

This makes me curious why ChisSoc had used their own fork instead of the fork from kevinvalk.

Since the dependency from ChisSoc has been deleted, we can't check if had updated some of the code. Let's hope were no major updates.

Reply:
I guess it was for compatibility, so further changes to the kevinvalk fork wouldn't break things in phoniebox, as it was always the latest commit used. Ironically it was now the case with his fork.
With the specific release in the original repo this is not a problem anymore.

Reply:
I will check if i have maybe an old installation somewhere on sd. Maybe there is a way to extract the files and compare them.

Reply:
unfortunately i dont have an old installation. Does someone of you maybe have one?
The installed files can be found at "/usr/local/lib/python3.9/dist-packages/pirc522" and copied for comparison.

Reply:
[pirc522.zip](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13198267/pirc522.zip)
Here is the content from a installation from June 4th. Hope it helps.

Reply:
> [pirc522.zip](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13198267/pirc522.zip) Here is the content from a installation from June 4th. Hope it helps.

was this a 2.x or 3.x installation? Looks like 2.x from the diff.

Reply:
Yes, 2.4.0. You needed 3.x? Then I'm sorry... 😢

Reply:
Yep, was about 3.x. 
But thanks anyway :)

Reply:
I like to pick-up this older discussion as I realized a difference when not using ChisSoc's version. My Mifare Ultralight C sticker do not work on a minimal hardware setup (battery powered RPi Zero 2) with the current rc522 package, but were fine with ChisSoc's version. I was unable to identify the actual issue but found that using the read_id function of the current rc522 package does the job properly. See my suggestion #2214.